source/jormungandr/jormungandr/scenarios/journey_filter.py

```diff
@@ -669,7 +686,9 @@ def is_fallback(section):
 
 
 def filter_similar_vj_journeys(journey_pairs_pool, request):
-    _filter_similar_journeys(journey_pairs_pool, request, similar_journeys_vj_generator)
+    _filter_similar_journeys(
+        journey_pairs_pool, request, similar_journeys_vj_generator, similar_bss_walking_vj_generator
```

Comment:
```suggestion
        journey_pairs_pool, request, (similar_journeys_vj_generator, similar_bss_walking_vj_generator)
```

Would it make sense to group the generators altogether (in a generator :smile:)  ?
That would avoid changing the `*similar_journey_generators` later down.

Reply:
I did this so that there would be minimum changes... :)